packages/sources/1forge/src/endpoint/quotes.ts

```diff
@@ -11,6 +11,9 @@ import { NAME as AdapterName } from '../config'
 export const supportedEndpoints = ['quotes', 'forex', 'price']
 export const batchablePropertyPath = [{ name: 'base' }, { name: 'quote' }]
 
+export const description =
+  'Returns a batched price comparison from a list currencies to a list of other currencies.\n\n[`/quotes`](https://1forge.com/api#quotes) - Convert from one currency to another.\n\n**NOTE: the `price` endpoint is temporarily still supported, however, is being deprecated. Please use the `quotes` endpoint instead.**'
+
```

Comment:
I'd recommend using backticks here and other descriptions with newlines, it'd make it cleaner imo

Reply:
How would you format this one for example? Here's my attempt at it:
```suggestion
export const description =
`Returns a batched price comparison from a list currencies to a list of other currencies.

[\`/quotes\`](https://1forge.com/api#quotes) - Convert from one currency to another.
**NOTE: the \`price\` endpoint is temporarily still supported, however, is being deprecated. Please use the \`quotes\` endpoint instead.**`
```

Reply:
Yeah, that's great imo 🙂

Reply:
Ok, all descriptions that used newline characters have been updated with this pattern